### PR TITLE
Mesh 900/protocol fee rpc

### PR DESCRIPTION
**Pull request by vkomenda**

### CODE REVIEW

File: pallets/protocol-fee/rpc/runtime-api/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_runtime::traits::{SaturatedConversion, UniqueSaturatedInto};
+
+/// A capped version of `Balance` which is normally a `u128`, fit into `u64` which is a serializable
+/// type unlike `u128`. There are no fees that would not fit into `u64`.
+#[derive(Eq, PartialEq, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
+#[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
+pub struct CappedFee(pub u64);
```
Comment (satyamakgec) [inline pallets/protocol-fee/rpc/runtime-api/src/lib.rs:17]: could you please add the rpc json in the schema

File: pallets/protocol-fee/rpc/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+/// Error type of this RPC API.
+pub enum Error {
+    /// The transaction was not decodable.
+    DecodeError = 1,
+    /// The call to runtime failed.
+    RuntimeError = 2,
+}
+
+impl From<Error> for i64 {
```
Comment (vkandy) [inline pallets/protocol-fee/rpc/src/lib.rs:41]: Minor but perhaps use C-like enums? In line 66: `ErrorCode::ServerError(Error::RuntimeError as i64)`

File: pallets/protocol-fee/rpc/runtime-api/src/lib.rs (hunk 1 above)
Comment (vkomenda) [reply pallets/protocol-fee/rpc/runtime-api/src/lib.rs:17]: Added.

File: pallets/protocol-fee/rpc/src/lib.rs (hunk 2 above)
Comment (vkomenda) [reply pallets/protocol-fee/rpc/src/lib.rs]: Done.

**Comment by vkomenda** [comment]
@satyamakgec please check if I updated the RPC schema OK.

File: polymesh_schema.json [hunk 3]
```diff
@@ -573,6 +574,19 @@
...
         },
+        "protocolFee": {
+            "computeFee": {
+                "description": "Gets the fee of a chargeable extrinsic operation",
+                "params": [
+                    {
+                        "name": "op",
+                        "type": "ProtocolOp",
+                        "isOptional": false
+                    }
```
Comment (satyamakgec) [inline polymesh_schema.json:585]: need to add the `at` as well as you can see in the other rpc definitions

Comment (vkomenda) [reply polymesh_schema.json:590]: Done, if you mean `blockHash`. It was indeed missing.

Comment (satyamakgec) [reply polymesh_schema.json:590]: Yeah

**Pull request merged**